### ISSUE TITLE
Notice URI pattern in Change notices

### ISSUE THREAD

**Issue by cristianvasquez** [OP-TED/ted-rdf-mapping-eforms#41]
Note: I'm marking is Issue with the label `URI patterns`, not part of the current mapping exercise but to track insights about URIs.

Change notice details:

HTML source: https://ted.europa.eu/en/notice/-/detail/126775-2024
XML source: https://ted.europa.eu/en/notice/126775-2024/xml

This notice modifies the contents of notice 739743-2023. The XML includes a reference to this notice in the following form:

`<efbc:ChangedNoticeIdentifier>739743-2023</efbc:ChangedNoticeIdentifier>`

However, notices are typically identified by URI patterns that use a UUID, such as:

`<cbc:ID schemeName="notice-id">98083d34-7771-4485-8ab4-9adc73cdb892</cbc:ID>`

This results in an `epo:ChangeInformation` that `epo:refersToPreviousNotice` to a value that will not exist.

![image](https://github.com/user-attachments/assets/0fccd2d7-f1d6-4814-93b6-d110ab7ffd1e)

This should be considered when defining the URI patterns (SHACL) policy.
- Note that notices also have a version number

**Comment by schivmeister** [comment]
I agree. However, this appears to be a data (eForms) issue, beyond the scope of mapping or even ePO. We will not be able to look up the actual notices, unless we have a sameAs table (or each notice is materialized with a sameAs to a URI w/ the document ID).

**Comment by csnyulas** [comment]
BTW, the problem is further complicated by missing, incomplete or wrong data, as explained in #112
